src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-block: 32px 16px;
+  padding-inline: 16px;
+
+  &__image {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+      background-image: url(/src/images/imac.jpeg);
+      background-size: 100% 100%;
```

Comment:
Using `background-size: 100% 100%;` will stretch the image to fit the container which might not preserve the aspect ratio. Consider using `background-size: cover;` to maintain the aspect ratio or provide a specific size that matches the original aspect ratio of the image.